.github/workflows/elixir.yml

```diff
@@ -16,8 +16,8 @@ jobs:
     steps:
       - uses: erlef/setup-beam@v1
         with:
-          elixir-version: 1.15
```

Comment:
What about using a matrix to test latest and older versions?

Reply:
if you know how to do it, that could be worthwhile in future